### DISCUSSION TITLE
Request: Film borders and sprockets; user controllable MTF curve.

### DISCUSSION THREAD

**Discussion by andreasabatelli** [videovillage/community#176] [Ideas]
Hi, would like to suggest the addition of film borders and sprockets, to get grain and halation on those elements, based on the parameters chosen from the user.
Would be even greater to get the duplicated previous/next frame, respectively above and below the hero frame, like if we were seeing the image from the scanner.

Also, would love to have the possibility to control the MTF curve, both for 16 and 35mm, to strenghten or loosen it at necessity.

**Comment by CC-Benno** [comment]
I Would love to see that too. :D
